### ISSUE TITLE
Add "Robot Account" feature like quay.io

### ISSUE THREAD

**Issue by devinus** [docker/hub-feedback#572]
Having to create a "fake" user for things like CI is experience--.

**Comment by edevil** [comment]
Exactly. A read-only account that we can include in deployment scripts without too much concern. And proper ACL support... Features that are already available on Docker Trusted Registry, but I need a hosted solution.

**Comment by ebr** [comment]
Is this on anyone's radar? i hate to "+1 comment", but it would be really nice to have this feature, especially for paying customers.

**Comment by VladislavRatslavCB** [comment]
With this feature, it's much easier to maintain and rotate credentials as needed.

**Comment by vdamery** [comment]
I think this feature match your request : https://docs.docker.com/enterprise/security/access-tokens/

Closing this issue due to inactivity.

Feel free to reopen it.
